tests/test_onnxml_imputer_converter.py

```diff
@@ -0,0 +1,110 @@
+"""
+Tests onnxml Imputer converter
+"""
+import unittest
+import warnings
+
+import numpy as np
+import torch
+from sklearn.impute import SimpleImputer
+
+from hummingbird.ml._utils import onnx_ml_tools_installed, onnx_runtime_installed, lightgbm_installed
+from hummingbird.ml import convert
+
+if onnx_runtime_installed():
+    import onnxruntime as ort
+if onnx_ml_tools_installed():
+    from onnxmltools import convert_sklearn
+    from onnxmltools.convert.common.data_types import FloatTensorType as FloatTensorType_onnx
+
+
+class TestONNXImputer(unittest.TestCase):
+    def _test_imputer_converter(self, norm, model=None):
```

Comment:
I think `norm` is never used.

Reply:
Thanks for catching that.  Those are a copy paste error from `normalizer`